src/main/java/examples/Examples.java

```diff
@@ -410,4 +410,24 @@ public void example12(MongoService mongoService) {
 
   }
 
+  public void example13_0(MongoService mongoService) {
+
+    JsonObject document = new JsonObject().put("title", "The Hobbit")
+        .put("publicationDate", new JsonObject().put("$date", -1018659600000L /* 21/9/1937 */));
```

Comment:
I think it would be clearer if the date wasn't a long literal. I imagine most users will deal with Java Date objects (and call getTime() on it)